tools/CMakeLists.txt

```diff
@@ -16,61 +16,17 @@
 #    Multiple authors (IBM Corp.) - initial implementation and documentation
 ###############################################################################
 
-if (NOT CMAKE_CROSSCOMPILING)
+include(OmrAssert)
 
-	add_subdirectory(hookgen)
-	add_subdirectory(tracemerge)
-	add_subdirectory(tracegen)
+omr_assert(TEST OMR_TOOLS)
+omr_assert(TEST NOT OMR_TOOLS_IMPORTFILE)
 
-	export(TARGETS hookgen tracemerge tracegen FILE "ImportTools.cmake")
+if(CMAKE_CROSSCOMPILING)
+	message(Status "OMR: Using cross compiled tools")
```

Comment:
Maybe a bit better message is in order. Had to do a double take on this one.  Not entirely sure how to make it better.  Maybe I just need more coffee